dowhy/causal_refuter.py

```diff
@@ -256,3 +257,66 @@ def __str__(self):
                 self.refutation_type, self.estimated_effect, self.new_effect, self.refutation_result['p_value']
             )
 
+class GraphRefutation(CausalRefutation):
```

Comment:
to be consistent, this class should be GraphRefuter and inherit CausalRefuter. 
CausalRefutation is the class that represents the output of a refuter class. 

Also this class should be inside causal_refuters folder, as a file of its own, "graph_refuter.py"